src/AddTodoForm.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "/vite.svg";
+
+function AddTodoForm() {
+  return (
+    <form>
+      <label htmlFor="todoTitle"> Title: </label>
+      <input id="todoTitle" type="text" />
```

Comment:
add a required attribute here so that a user is required to add an item in the input before submitting

```suggestion
      <input id="todoTitle" type="text" required />
```